interview-questions.md

```diff
@@ -30,34 +30,58 @@ Researched answer: the method .map() returns an array of informatin from the ori
 
 answer via: https://www.google.com/search?q=What+is+the+difference+between+map+and+filter%3F&oq=What+is+the+difference+between+map+and+filter%3F&aqs=chrome..69i57j0i512j0i20i263i512j0i512l2j0i22i30l5.644j0j15&sourceid=chrome&ie=UTF-8
 
-1. What is iteration?
+4. What is iteration?
 
-Your answer:
+Your answer: iteration is an action being performed continuoulsy until the goal is reached. For example in programming there are methods that can be used to accomplish iteration. The .map() method will iterate over an array until all index values have been accounted for. 
 
-Researched answer:
+Researched answer: a sequence of code that is repeated until the desired result is completed.
 
-1. What is the difference between a function and a method?
+answer via: https://www.techtarget.com/searchsoftwarequality/definition/iterative#:~:text=In%20programming%20specifically%2C%20iterative%20refers,called%20circular%20or%20evolutionary%20development.
+
+5. What is the difference between a function and a method?
+
+Your answer: A function is a developer made variable that contains parameters that when used properly exectue some form of logic. A method can either be built in like .filter() or made for specific instances. Methods are what can used to perform actions in classes. 
+
+Researched answer: Functions are instructions that perform a specfic task. Methods are instruciton that are to be performed when using objects. 
+
+answer via: https://www.educative.io/answers/what-is-the-difference-between-a-method-and-a-function
 
-Your answer:
 
-Researched answer:
 
 6. STRETCH: What is hoisting in JavaScript?
 
-Your answer:
+Your answer: Hoisting is when functions are used to extract elements from classes for tasks. 
 
-Researched answer:
+Researched answer: the process of moving functions, variables, and classes to the top of the scope. This is before the execution of the code.
+
+JavaScript Hoisting refers to the process whereby the interpreter appears to move the declaration of functions, variables or classes to the top of their scope, prior to execution of the code.
```

Comment:
This is some good research, way to stretch yourself! For your next assessment, remember we love to hear your own interpretation of what you've learned. Keep up the good work! 🙂